warehouse/schema.go

```diff
@@ -22,36 +22,38 @@ type SchemaHandleT struct {
 	uploadSchema                  warehouseutils.SchemaT
 }
 
-func HandleSchemaChange(existingDataType, columnType string, columnVal interface{}) (newColumnVal interface{}, ok bool) {
+func HandleSchemaChange(existingDataType, newDataType string, value interface{}) (interface{}, bool) {
```

Comment:
[minor] go version 1.18  introduced `any` as an alias to `interface{}`. I personally, find it prettier.
```suggestion
func HandleSchemaChange(existingDataType, newDataType string, value any) (any, bool) {
```

Reply:
Done.